### PR TITLE
add form actions to command palette

### PR DESCRIPTION
**Pull request by christianguenter2**
Now form actions are available on command palette too.

E.g. commit page
grafik

E.g. settings page
grafik

### CODE REVIEW

File: src/ui/zcl_abapgit_html_form.clas.abap [hunk 1]
```diff
@@ -758,7 +758,8 @@ CLASS zcl_abapgit_html_form IMPLEMENTATION.
     IF is_field-side_action IS NOT INITIAL.
       ii_html->add( '</div>' ).
       ii_html->add( '<div class="command-container">' ).
-      ii_html->add( |<input type="submit" value="&#x2026;" formaction="sapevent:{ is_field-side_action }">| ).
+      ii_html->add( |<input type="submit" value="&#x2026;" formaction="sapevent:{ is_field-side_action }"|
+                 && |title="{ is_field-label }">| ).
```
Comment (mbtools) [inline src/ui/zcl_abapgit_html_form.clas.abap:762]: might need a space before title

Comment (christianguenter2) [reply src/ui/zcl_abapgit_html_form.clas.abap]: Thx. Fixed.

**Comment by mbtools** [comment]
Works well 👍 

Only "Help" button is missing, which is an href but not in the toolbar:

```html
<a class="dialog-help" href="sapevent:url?url=https://docs.abapgit.org/guide-settings-global.html" title="Help"><i class="icon icon-question-circle-solid"></i></a>
```

**Comment by christianguenter2** [comment]
Yeah, I try to address that and others in follow up PRs :-)

**Pull request merged**